hardened-chromium-source.spec

```diff
@@ -0,0 +1,59 @@
+%global numjobs %{_smp_build_ncpus}
+
+Name:	 hardened-chromium-source
```

Comment:
since this technically has little to do with the hardened-chromium changes, I wonder if we should just call this `chromium-source-clean`?

Otherwise it kind of reminds me of gentoo's kernel-source package where it contains the source for the mainline kernel, whereas this package isn't going to have the complete hardened-chromium source, if you get what I'm saying